$NetBSD: patch-CMakeLists.txt,v 1.4 2021/11/02 18:35:04 adam Exp $

Do not look for X11 on macOS.
Do not bother with uninstaller.

--- CMakeLists.txt.orig	2021-10-24 00:23:51.000000000 +0000
+++ CMakeLists.txt
@@ -76,7 +76,9 @@ find_package(ICU REQUIRED COMPONENTS dat
 # no, gettext executables are not required when NLS is deactivated
 find_package(Gettext)
 
+if(NOT APPLE)
 find_package(X11)
+endif(NOT APPLE)
 
 if(NOT WIN32)
 	# needed to get some SDL2 defines in... (as of rev31694 -D_GNU_SOURCE=1 is required!)
@@ -568,11 +570,6 @@ endif()
 # uninstall
 #
 
-configure_file(
-	"${CMAKE_SOURCE_DIR}/cmake/uninstall.cmake.in"
-	"${CMAKE_CURRENT_BINARY_DIR}/uninstall.cmake"
-	IMMEDIATE @ONLY
-)
 
 add_custom_target(uninstall
 	"${CMAKE_COMMAND}" -P "${CMAKE_CURRENT_BINARY_DIR}/uninstall.cmake"
